.vscode/settings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.organizeImports": true
```

Comment:
Thanks, this was driving my insane! 
Should we run this on all files in this PR? Other PR's editing existing files will have a lot of changes unrelated to the PR

Reply:
Sure I'll get on that

Reply:
Great thanks! Anyone who rebases will probably have simple merge conflicts as a result.